core/init.go

```diff
@@ -13,11 +13,12 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/utils"
 )
 
-// defaultValKeyType is default crypto algo for consensus in core
+// defaultValKeyType is a default crypto algo for consensus in core
 const defaultValKeyType = types.ABCIPubKeyTypeSecp256k1
 
-// Init initializes the Core repository under the given 'path'.
-// It checks and creates the default if not exist:
+// It(func Init) checks each item of the list(below) for existence in the path.
```

Comment:
```suggestion
// Init checks each item of the list(below) for existence in the path.
```

This is necessary for godocs